### PR TITLE
Update all modules

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
bot, build

**Comment by Alexander-Wilms** [comment]
bot, build

**Comment by Alexander-Wilms** [comment]
bot, build

**Comment by Alexander-Wilms** [comment]
bot, build

File: io.elementary.code.json [hunk 1]
```diff
@@ -65,6 +61,7 @@
 			"buildsystem": "cmake",
 			"config-opts": [
 				"-DCMAKE_BUILD_TYPE=Release",
+				"-DCMAKE_INSTALL_PREFIX=/app",
```
Comment (TingPing) [inline io.elementary.code.json:64]: This is not necessary.

Comment (Alexander-Wilms) [reply io.elementary.code.json]: Fixed

**Pull request merged**